doc/userDocs/usersGuide.md

```diff
@@ -1356,4 +1409,79 @@ As a result, when using node labels it is recommended that you label all of your
 worker nodes with either `wvrole=driver` or `wvrole=sut`.  This will give you the 
 best control over pod placement.
 
+## Running on Specific Cluster Types<a name="specific"></a>
+
+### Running on Minikube
+
+This section discusses requirements for running Weathervane on a laptop or workstation using a 
+Kubernetes cluster created with [Minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+#### Cluster Requirements
+
+In order to run Weathervane with a single Micro instance, you must create your Minikube 
+cluster with at least 3 cpus and 8GB of memory.  In addition, the default-storageclass 
+addon must be enabled (this is the default).  The micro instance will also use 15.25GB 
+of disk space.
+
+The command to start Minikube with 3 cpus and 8Gb is :
+* `minikube start --cpus=3 --memory=8g`
+
+Note that even with this cluster configuration, Minikube may have trouble running Weathervane 
+depending on how much memory is configured on the laptop and what else is 
+running.
+
+#### Weathervane Configuration
+
+In order to run Weathervane on Minikube, you need to use the following configuration 
+parameter values:
+
+| Parameter | Value |
+|-----------|-------|
+| appIngressMethod | clusterIp |
```

Comment:
should be 'clusterip' ? (lower case i)

Reply:
yes.  I just pushed a commit with fixes for these comments.